docker-compose.yml

```
@@ -49,10 +49,12 @@ services:
       LANGFUSE_HOST: "http://langfuse:3000" # Define the host for langfuse
       LANGFUSE_PUBLIC_KEY: "pk-lf-1234567890" # Define the public key for langfuse
       LANGFUSE_SECRET_KEY: "sk-lf-1234567890" # Define the secret key for langfuse
+      LANGFUSE_ENVIRONMENT: "local"
       LANGFUSE_RELEASE: "local"
       # LANGFUSE_DEBUG: "True"
       LANGFUSE_TIMEOUT: "10"
       LANGFUSE_SAMPLE_RATE: "1.0"
+      LANGFUSE_FORCE_RUN: "True"
```

Shouldn't force run true in compose, should be commented out probably before merge?

---

it should! This was from my local setup.

```suggestion
      # LANGFUSE_FORCE_RUN: "True"
```